backend/src/main/java/ca/bc/gov/hlth/hnweb/converter/rapid/R41Converter.java

```diff
@@ -1,48 +1,95 @@
 package ca.bc.gov.hlth.hnweb.converter.rapid;
 
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
 import org.apache.commons.lang3.StringUtils;
 
-import ca.bc.gov.hlth.hnweb.model.InquirePhnRequest;
-import ca.bc.gov.hlth.hnweb.model.InquirePhnResponse;
-import ca.bc.gov.hlth.hnweb.model.rapid.CI0;
-import ca.bc.gov.hlth.hnweb.model.rapid.RapidDefaults;
+import ca.bc.gov.hlth.hnweb.model.StatusEnum;
+import ca.bc.gov.hlth.hnweb.model.eligibility.InquirePhnBeneficiary;
+import ca.bc.gov.hlth.hnweb.model.eligibility.InquirePhnRequest;
+import ca.bc.gov.hlth.hnweb.model.eligibility.InquirePhnResponse;
 import ca.bc.gov.hlth.hnweb.model.rapid.RPBSHeader;
 import ca.bc.gov.hlth.hnweb.model.rapid.RPBSPPE0;
+import ca.bc.gov.hlth.hnweb.model.rapid.RPBSPPE0Beneficiary;
+import ca.bc.gov.hlth.hnweb.model.rapid.RPBSPPE0Data;
 
 public class R41Converter extends BaseRapidConverter {
```

Comment:
Yeah. I just named it R41 out of familiarity. But maybe we can standardize on naming them according to the downstream format? Because that's where the logic applies. So for the V2 ones we can still use R15Converter etc. I'd be more inclined to name this RPBSPPE0Converter since that's what we're creating.

Reply:
Also, it sounds like we might have additional refactoring to do so I will hold off on R42 until then.